udata/sentry.py

```diff
@@ -11,18 +12,26 @@
 log = logging.getLogger(__name__)
 
 RE_DSN = re.compile(
-    r'(?P<scheme>https?)://(?P<client_id>[0-9a-f]+):[0-9a-f]+'
-    '@(?P<domain>.+)/(?P<site_id>\d+)')
+    r'(?P<scheme>https?)://(?P<client_id>[0-9a-f]+)(?::(?P<secret>[0-9a-f]+))?'
+    r'@(?P<domain>.+)/(?P<site_id>\d+)')
+
+SECRET_DSN_DEPRECATED_MSG = 'DSN with secret is deprecated, use a public DSN instead'
 
 # Controlled exceptions that Sentry should ignore
 IGNORED_EXCEPTIONS = HTTPException, PermissionDenied, UploadProgress
 
-
 def public_dsn(dsn):
-    '''Transform a standard Sentry DSN into a public one'''
+    '''Check if DSN is public or raise a warning and turn it into a public one'''
     m = RE_DSN.match(dsn)
     if not m:
         log.error('Unable to parse Sentry DSN')
```

Comment:
not having a return or raise here is strange, don't you think (I know it's not your code)

Reply:
agreed, I'll change it!